srv/pillar/ceph/disk_led.sls

```diff
@@ -0,0 +1,15 @@
+# This is the default configuration for the storage enclosure LED blinking.
```

Comment:
This file needs to be installed and registered (deepsea.spec.in and Makefile)

Reply:
Done